client/state/data-layer/wpcom/read/following/mine/index.js

```diff
@@ -19,6 +19,19 @@ import { errorNotice } from 'state/notices/actions';
 const ITEMS_PER_PAGE = 200;
 const MAX_ITEMS = 2000;
 
+export function toValidId( val ) {
```

Comment:
> it's unclear to me if webpack2 will fix it with tree shaking.

often I get the impression like we think in the JS world that webpack2 and tree shaking will miraculously solve our every size problem. I just don't see us benefitting much from it at all, especially since we don't write many things that don't get imported by other modules…